supabase/migrations/20250203140131_add-type-specific-details-to-accounts.sql

```diff
@@ -0,0 +1,3 @@
+alter table "wallet"."accounts" add column "details" jsonb not null;
```

Comment:
as you can see instead of adding type specific columns on the row I added one json column so we can put type specifc data there. so cashu account would have `{ mintUrl: 'some-url' }` stored there while nwc would have `{ nwcUrl: 'some-url' }`